python/lookout/sdk/service_analyzer.py

```diff
@@ -23,8 +23,9 @@ def __new__(cls, clsname, bases, dct):
             if name.startswith("__"):
                 continue
 
+            default_func = dct.get(name, func)
```

Comment:
I knew it's 1 line change!

Reply:
yeah I was ignoring if the method is actually defined 😅.